src/whisper_transcriber/transcriber.py

```
@@ -0,0 +1,241 @@
+        
+        if language:
+            options["language"] = language
+            
+        with Progress(
+            SpinnerColumn(),
+            TextColumn("[progress.description]{task.description}"),
+            BarColumn(),
+            TimeElapsedColumn(),
+            console=console,
+        ) as progress:
+            task_id = progress.add_task("Transcribing audio...", total=None)
```

Ruff (0.12.2)

Local variable `task_id` is assigned to but never used

Remove assignment to unused variable `task_id`

(F841)